In [2]:
import numpy as np

1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

$\overline{X}$ = 80;       $\sigma$ = 16;        $n$ = 256;      $Z_{\alpha/2}$ = 1.96

$$\overline{X} \pm Z_{\alpha/2} * \frac{\sigma}{\sqrt n}$$

отсюда получаем доверительный интервал:$$80 \pm 1.96 * \frac{16}{\sqrt 256}=80 \pm 1.96 * \frac{16}{16}=[78.04; 81.96]$$

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95.

Из полученных опытных данных находим $\overline{X}$ и $\sigma$

In [10]:
z = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
mean = sum(z)/len(z)
std = np.sqrt(sum((x - mean)**2 for x in z) / (len(z)-1))
print(f'X = {mean: .2f};  std = {std: .2f}')

X =  6.59;  std =  0.45


$\overline{X}$ = 6.59;       $\sigma$ = 0.45;        $n$ = 10; по таблице находим $t_{\alpha/2}$ = 2.262

$$\overline{X} \pm t_{\alpha/2} * \frac{\sigma}{\sqrt n}$$

отсюда получаем доверительный интервал:$$6.59 \pm 2.262 * \frac{0.45}{\sqrt 10}=6.59 \pm 0.32=[6.27; 6.91]$$

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с =0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

H0: диаметр = 17 мм, $\mu = \mu 0$

H1: диаметр = 17.5 мм, $\mu > \mu 0$

$\sigma^2 = D = 4; \sigma = 2$

по даблице находим $Z_T = 1.645$ $$Z_H = \frac {\overline{X} - \mu 0} {\sigma / \sqrt n} = \frac{17.5-17}{2/\sqrt 100} = \frac{0.5}{0.2} = 2.5$$

получили, что $Z_T = 1.645 < Z_H = 2.5$ Верна Н1

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [11]:
z = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
mean = sum(z)/len(z)
std = np.sqrt(sum((x - mean)**2 for x in z) / (len(z)-1))
print(f'X = {mean: .2f};  std = {std: .2f}')

X =  198.50;  std =  4.45


H0: вес пачки печенья = 200 г, $\mu = \mu 0$

H1: вес пачки печенья = 198.5 г, $\mu < \mu 0$

$\sigma = 4.45$

Т.к. достоверная вероятность равна 99%, значит $\alpha$ = 1%, по даблице находим $Z_T = -2.33$
$$Z_H = \frac {\overline{X} - \mu 0} {\sigma / \sqrt n} = \frac{198.5-200}{4.45/\sqrt 10} = -\frac{1.5}{1.41} = -1.1$$

получили, что $Z_T = -2.33 < Z_H = -1.1$ Верна Н0